sql/core/src/test/resources/sql-tests/results/group-by-ordinal.sql.out

```diff
@@ -141,12 +141,12 @@ from
 -- !query 13 schema
 struct<a:int,rand(0):double,sum(b):bigint>
 -- !query 13 output
-1	0.4048454303385226	2
-1	0.8446490682263027	1
-2	0.5871875724155838	1
-2	0.8865128837019473	2
-3	0.742083829230211	1
-3	0.9179913208300406	2
+1	0.5234194256885571	2
```

Comment:
Same comment here, but, I suppose we have to assert the exact output given that this is an exact-match test. This is OK. In that case I think it's all the more reasonable to fix the random tests above.